starter-code/scripts/controllers/routes.js

```diff
@@ -1,6 +1,22 @@
-/* TODO: Configure our two routes for this app with page.js,
+/* TODO DONE: Configure our two routes for this app with page.js,
     by registering each URL your app can handle, linked to a
     single controller function to handle it: */
 
-/* TODO: What function do we call to activate page.js?
+$(document).ready(function(){
+	page();
+});
+page('/' , article);
+page('/about', about);
+
+function article() {
+  articleController.reveal();
+  console.log('article() ran');
```

Comment:
It's great to console log to verify if the target has been hit. Try to remove all console logs before you submit the assignment (or before submitting to production).